atom/browser/api/atom_api_desktop_capturer.cc

```diff
@@ -62,7 +62,11 @@ DesktopCapturer::DesktopCapturer(v8::Isolate* isolate) {
   Init(isolate);
 }
 
-DesktopCapturer::~DesktopCapturer() {}
+DesktopCapturer::~DesktopCapturer() {
+  window_capturer_.reset();
```

Comment:
Are these resets necessary? Won't this happen automatically?

Reply:
yup they will be cleaned up automatically.

Reply:
This was me puzzling over some memory leaks before I dove into the refactor, I'll remove